src/env.ts

```diff
@@ -6,6 +6,8 @@ export const env = {
   EMILY_URL: process.env.EMILY_URL,
   MEMPOOL_API_URL:
     process.env.MEMPOOL_API_URL || "http://localhost:8083/api/v1/",
+  MEMPOOL_URL: process.env.MEMPOOL_URL,
```

Comment:
Can't we just rely on a single env variable for MEMPOOL and derive the other from it?